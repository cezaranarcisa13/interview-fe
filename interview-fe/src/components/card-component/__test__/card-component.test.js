import React from "react";
import ReactDOM from "react-dom";
import CardComponent from "../card-component";
import renderer from "react-test-renderer";

it("renders witout crashing", () => {
    const div = document.createElement("div");
    ReactDOM.render(<CardComponent />, div);
})

it("match snapshot", () => {
    const info = {
        message: "Buna",
        user: {
            name: "Narcisa",
            last: "Petcu"
        }
    };
    const tree = renderer.create(<CardComponent info={info} />).toJSON();
    expect(tree).toMatchSnapshot();
})