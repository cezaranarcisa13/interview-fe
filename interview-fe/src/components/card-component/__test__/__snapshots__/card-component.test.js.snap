// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`match snapshot 1`] = `
<div
  className="card"
>
  <div
    className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-bhp9pd-MuiPaper-root-MuiCard-root"
  >
    <div
      className="MuiCardContent-root css-46bh2p-MuiCardContent-root"
    >
      <p
        className="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom css-1ydyjjr-MuiTypography-root"
      >
        Message card
      </p>
      <p
        className="MuiTypography-root MuiTypography-body2 css-e784if-MuiTypography-root"
      >
        Buna
      </p>
      <p
        className="MuiTypography-root MuiTypography-body1 css-1pnmrwp-MuiTypography-root"
      >
        For user
      </p>
      <p
        className="MuiTypography-root MuiTypography-body2 css-e784if-MuiTypography-root"
      >
        Narcisa  Petcu
      </p>
    </div>
  </div>
</div>
`;
